.version

```diff
@@ -1 +1 @@
-2017.11.01
```

Comment:
One final comment here -- since this release is based on Snabb v2018.01 it should be Snabb lwAFTR v2018.01.XXXX, not .03.  Can we change to v2018.01.2.1 or something?  I know that having the four version components is not great but given that upstream had to make a .1 and a .2 release, it seems reasonable.